metricbeat/module/system/network/_beat/docs.asciidoc

```diff
@@ -0,0 +1,23 @@
+=== System Network MetricSet
+
+The System Network MetricSet provides network IO metrics collected from the
+operating system. One event is created for each network interface.
+
+By default metrics are reported from all network interfaces. To select which
+interfaces that metrics are reported from, use the `interfaces` configuration
+options. The value must be an array of interface names. For example:
+
+[source,yaml]
+----------------------------
+metricbeat.modules:
+- module: system
+  metricsets: [network]
```

Comment:
In most other examples we use `metricset: ["network"]`. I'm happy with both. Which one should we pick?